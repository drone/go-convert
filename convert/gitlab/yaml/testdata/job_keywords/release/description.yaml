# https://docs.gitlab.com/ee/ci/yaml/#releasedescription

job:
  release:
    tag_name: ${MAJOR}_${MINOR}_${REVISION}
    description: './path/to/CHANGELOG.md'