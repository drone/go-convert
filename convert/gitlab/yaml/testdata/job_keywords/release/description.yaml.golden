job:
    release:
        tag_name: ${MAJOR}_${MINOR}_${REVISION}
        description: ./path/to/CHANGELOG.md
